<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qSlicerSessionDataRecorderModule</class>
 <widget class="qSlicerWidget" name="qSlicerSessionDataRecorderModule">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>488</width>
    <height>751</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>345</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="1">
    <widget class="ctkCollapsibleButton" name="CollapsibleButton_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>225</height>
      </size>
     </property>
     <property name="text">
      <string>Log in </string>
     </property>
     <property name="contentsFrameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <widget class="QGroupBox" name="groupBoxAuthentication">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>341</width>
        <height>171</height>
       </rect>
      </property>
      <property name="title">
       <string>Authentication</string>
      </property>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>151</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Password:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEditUsername">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>50</y>
         <width>211</width>
         <height>21</height>
        </rect>
       </property>
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Normal</enum>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>151</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Username:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEditPassword">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>80</y>
         <width>211</width>
         <height>21</height>
        </rect>
       </property>
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonLogin">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>110</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Login</string>
       </property>
       <property name="shortcut">
        <string>Return</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
       <property name="default">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonCreate">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>140</y>
         <width>101</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Create New User</string>
       </property>
      </widget>
      <zorder>label_2</zorder>
      <zorder>lineEditUsername</zorder>
      <zorder>label_3</zorder>
      <zorder>lineEditPassword</zorder>
      <zorder>pushButtonLogin</zorder>
      <zorder>pushButtonCreate</zorder>
      <zorder>CollapsibleButton_2</zorder>
     </widget>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="ctkCollapsibleButton" name="HelloCollapsibleButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Save Session</string>
     </property>
     <property name="contentsFrameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>431</width>
        <height>221</height>
       </rect>
      </property>
      <property name="title">
       <string>Saving Trainee Data</string>
      </property>
      <widget class="QLabel" name="MessageLabel">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>291</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>1) Enter Student ID and click OK. </string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Student ID *:</string>
       </property>
      </widget>
      <widget class="QGroupBox" name="groupBoxTraineeData">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>381</width>
         <height>81</height>
        </rect>
       </property>
       <property name="title">
        <string>Additional Trainee Data</string>
       </property>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>151</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>First name:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>30</y>
          <width>91</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Last name:</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEditFirstName">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEditLastName">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>50</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="saveButton">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>50</y>
          <width>101</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Create Trainee</string>
        </property>
       </widget>
       <zorder>label</zorder>
       <zorder>label_6</zorder>
       <zorder>lineEditFirstName</zorder>
       <zorder>lineEditLastName</zorder>
       <zorder>saveButton</zorder>
       <zorder>groupBoxAuthentication</zorder>
      </widget>
      <widget class="QLineEdit" name="lineEditStudentID">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>40</y>
         <width>171</width>
         <height>21</height>
        </rect>
       </property>
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="OkButton">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>40</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>180</y>
         <width>301</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>2) At the end of the user's session click:  &quot;Save Scene&quot;</string>
       </property>
      </widget>
      <widget class="QPushButton" name="saveSceneButton">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>180</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Save Scene</string>
       </property>
      </widget>
     </widget>
     <zorder>groupBox</zorder>
     <zorder>CollapsibleButton</zorder>
     <zorder>CollapsibleButton</zorder>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="ctkCollapsibleButton" name="CollapsibleButton">
     <property name="text">
      <string>Load Session</string>
     </property>
     <property name="contentsFrameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qSlicerWidget</class>
   <extends>QWidget</extends>
   <header>qSlicerWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkCollapsibleButton</class>
   <extends>QWidget</extends>
   <header>ctkCollapsibleButton.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
